bin/install.sh

```diff
@@ -33,7 +33,7 @@ if [ "$WEB_UPGRADE" = false ]; then
 
   # Make sure the command is launched interactive.
   if ! [ -t 0  ]; then
-    echo -e "Detected old installation command. Please use:\n$ bash <(curl -sL https://www.screenlyapp.com/install-ose.sh)"
+    echo -e "Detected old installation command. Please use:\n$ bash <(curl -sL https://install-ose.srly.io)"
```

Comment:
I set up a new URL instead, which is `https://install-anthias.srly.io`. Could you update it?

Reply:
Sure.